desktop/core/src/desktop/settings.py

```diff
@@ -654,23 +654,30 @@ def is_oidc_configured():
 ################################################################
 
 # Insert our custom upload handlers
-file_upload_handlers = [
+if get_chunked_fileuploader():
```

Comment:
The config is self explanatory. Let's use that here directly instead of creating a new method and calling it there?

Reply:
Added explanation in a similar comment. Resolving it.